packages/manager/apps/web/client/app/private-database/private-database.service.js

```diff
@@ -1332,4 +1334,22 @@ export default class PrivateDatabase {
       },
     });
   }
+
+  translate(keyToTranslate, { versionNumber }) {
```

Comment:
this method should not belong to private-database.service

i don't think you need to create a method just use $translate.instant where it's need to be called